src/backend/distributed/sql/citus--12.0-1--12.1-1.sql

```diff
@@ -1,5 +1,9 @@
 -- citus--12.0-1--12.1-1
 
+#include "udfs/citus_pause_node_within_txn/12.1-1.sql"
+
+
+
 -- bump version to 12.1-1
 
 #include "udfs/citus_prepare_pg_upgrade/12.1-1.sql"
```

Comment:
How about adding the new line down below?

```suggestion
-- bump version to 12.1-1

#include "udfs/citus_pause_node_within_txn/12.1-1.sql"
#include "udfs/citus_prepare_pg_upgrade/12.1-1.sql"
```